sdk/compute/armcompute/CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Release History
 
+## v0.2.1 (released)
+No exported changes
```

Comment:
We have a lot of changes on the return value of functions, we should have plenty of breaking changes.